analyses/rna-seq-protocol-dge/README.md

```diff
@@ -0,0 +1,184 @@
+
+Evaluate the p-value distribution from first-pass DGE analysis, which will likely exhibit rightward skew.
+
+Select genes that are stably expressed in poly-A and stranded RNA-seq libraries as empirical negative control genes for use in other DGE analyses varying by library enrichment method.
+
+### Methods
+
+1. Collapse `pbta-gene-counts-rsem-expected_count` of stranded and poly-A libraries using the OpenPBTA-analysis/analyses/collapse-rnaseq module.
+2. Select `Kids_First_Biospecimen`s with the same `sample_id`s and both stranded and poly-A `RNA_library`s.
+    Index | Kids_First_Biospecimen_ID | sample_id | experimental_strategy | RNA_library | cohort
+    ------|---------------------------|-----------|-----------------------|-------------|--------
+    1     | BS_HE0WJRW6               | 7316-1455 | RNA-Seq               | stranded    | CBTN
+    2     | BS_HWGWYCY7               | 7316-1455 | RNA-Seq               | poly-A      | CBTN
+    3     | BS_SHJA4MR0               | 7316-161  | RNA-Seq               | stranded    | CBTN
+    4     | BS_X0XXN9BK               | 7316-161  | RNA-Seq               | poly-A      | CBTN
+    5     | BS_FN07P04C               | 7316-255  | RNA-Seq               | stranded    | CBTN
+    6     | BS_W4H1D4Y6               | 7316-255  | RNA-Seq               | poly-A      | CBTN
+    7     | BS_8QB4S4VA               | 7316-536  | RNA-Seq               | stranded    | CBTN
+    8     | BS_QKT3TJVK               | 7316-536  | RNA-Seq               | poly-A      | CBTN
+    9     | BS_7WM3MNZ0               | A16915    | RNA-Seq               | poly-A      | PNOC003
+    10    | BS_KABQQA0T               | A16915    | RNA-Seq               | stranded    | PNOC003
+    11    | BS_68KX6A42               | A18777    | RNA-Seq               | poly-A      | PNOC003
+    12    | BS_D7XRFE0R               | A18777    | RNA-Seq               | stranded    | PNOC003
+3. Run edgeR `exactTest`, edgeR `glmLRT`, edgeR `glmQLFTest`, and DESeq2 `nbinomWaldTest` comparing stranded and poly-A RNA-seq `rsem-expected_count`s, using an adapted workflow from the [OMPARE project](https://github.com/d3b-center/OMPARE/blob/master/code/patient_level_analyses/utils/rnaseq_edger_normalizations.R#L37-L41). RSEM expected read counts are normalized using either UQ-pgQ2 or TMM. The UQ-pgQ2 normalization method has lower false positive rate in DGE testing (<https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6502-7>).
+4. Select genes that are stably expressed in poly-A and stranded RNA-seq libraries by the following criteria, and the filtered output `results/stranded_vs_polya_stably_exp_genes.csv` is ordered by the coefficient of variations.
```

Comment:
should it be results/NORMALIZATION_METHOD/stranded_vs_polya_stably_exp_genes.csv ?

```suggestion
4. Select genes that are stably expressed in poly-A and stranded RNA-seq libraries by the following criteria, and the filtered output `results/NORMALIZATION_METHOD/stranded_vs_polya_stably_exp_genes.csv` is ordered by the coefficient of variations. 
```

Reply:
Yes. Thank you for catching it.